packages/eslint-plugin/src/rules/explicit-member-accessibility.ts

```diff
@@ -119,30 +119,28 @@ export default util.createRule<Options, MessageIds>({
         return;
       }
 
-      if (util.isTypeScriptFile(context.getFilename())) {
-        // const methodName = util.getNameFromPropertyName(methodDefinition.key);
-        const methodName = util.getNameFromClassMember(
+      // const methodName = util.getNameFromPropertyName(methodDefinition.key);
```

Comment:
Please remove the commented out code

Reply:
Just seen that you didn't add this, it was already there, but if you wouldn't mind cleaning it up anyway that would be great!

Reply:
@JamesHenry Fix from 601159d!